### ISSUE TITLE
[release/2.11] timm_models_xcit_large_24_p8_224_float16_training accuracy test failed on PTL Windows

### ISSUE THREAD

**Issue by shangerxin** [intel/torch-xpu-ops#2960]
### 🐛 Describe the bug

BMG Windows also has the same issue https://github.com/intel/torch-xpu-ops/issues/2924.

# Log 
`

loading model: 0it [00:00, ?it/s]
loading model: 0it [00:09, ?it/s]
xpu  train xcit_large_24_p8_224               
WARNING:common:fp64 golden ref were not generated for xcit_large_24_p8_224. Setting accuracy check to cosine
W0227 00:30:34.207000 7840 site-packages\torch\_inductor\utils.py:1731] [2/0_1] Not enough SMs to use max_autotune_gemm mode
W0227 00:33:56.256000 7840 site-packages\torch\_dynamo\utils.py:3303] Similarity score=0.9740583896636963
E0227 00:33:56.258000 7840 site-packages\torch\_dynamo\utils.py:3231] Accuracy failed for key name blocks.16.local_mp.conv2.weight.grad
fail_accuracy
`

# Reproduce
`
echo "Info:  >>>>>>          set http_proxy=http://proxy.ims.intel.com:911         set https_proxy=http://proxy.ims.intel.com:911         call conda activate "C:/Jenkins/workspace/PTL_Timms_Dynamo_Benchmark\v"         call "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\Build\vcvarsall.bat" x64         python C:\Jenkins\workspace\PTL_Timms_Dynamo_Benchmark\qa\new-products\script\clean_whitespace.py C:\Jenkins\workspace\PTL_Timms_Dynamo_Benchmark\artifacts\run_timm_benchmark_res2net50_14w_8s.bat         "

set http_proxy=http://proxy.ims.intel.com:911
set https_proxy=http://proxy.ims.intel.com:911
call conda activate "C:/Jenkins/workspace/PTL_Timms_Dynamo_Benchmark\v"
call "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\Build\vcvarsall.bat" x64
python C:\Jenkins\workspace\PTL_Timms_Dynamo_Benchmark\qa\new-products\script\clean_whitespace.py C:\Jenkins\workspace\PTL_Timms_Dynamo_Benchmark\artifacts\run_timm_benchmark_res2net50_14w_8s.bat

echo "Info:  >>>>>> Running benchmark timms model xcit_large_24_p8_224 with command:  pushd C:\Jenkins\workspace\PTL_Timms_Dynamo_Benchmark\pytorch pip list > "C:\Jenkins\workspace\PTL_Timms_Dynamo_Benchmark\artifacts\timms_pip_list.log" 2>&1 set ZE_PATH=C:\level-zero-win-sdk-1.20.2 python benchmarks\dynamo\timm_models.py --accuracy -d xpu -n10 --backend=inductor --cold-start-latency --training  --float16 --only xcit_large_24_p8_224  --output=C:\Jenkins\workspace\PTL_Timms_Dynamo_Benchmark\artifacts\0\inductor_timm_models_xcit_large_24_p8_224_float16_training_xpu_accuracy.csv >C:\Jenkins\workspace\PTL_Timms_Dynamo_Benchmark\artifacts\0\inductor_timm_models_xcit_large_24_p8_224_float16_training_xpu_accuracy.log 2>&1 popd                     "
echo "Info:  >>>>>> Running timms benchmark iteration: 1/1"
echo "Info:  >>>>>>          set http_proxy=http://proxy.ims.intel.com:911         set https_proxy=http://proxy.ims.intel.com:911         call conda activate "C:/Jenkins/workspace/PTL_Timms_Dynamo_Benchmark\v"         call "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\Build\vcvarsall.bat" x64          pushd C:\Jenkins\workspace\PTL_Timms_Dynamo_Benchmark\pytorch pip list > "C:\Jenkins\workspace\PTL_Timms_Dynamo_Benchmark\artifacts\timms_pip_list.log" 2>&1 set ZE_PATH=C:\level-zero-win-sdk-1.20.2 python benchmarks\dynamo\timm_models.py --accuracy -d xpu -n10 --backend=inductor --cold-start-latency --training  --float16 --only xcit_large_24_p8_224  --output=C:\Jenkins\workspace\PTL_Timms_Dynamo_Benchmark\artifacts\0\inductor_timm_models_xcit_large_24_p8_224_float16_training_xpu_accuracy.csv >C:\Jenkins\workspace\PTL_Timms_Dynamo_Benchmark\artifacts\0\inductor_timm_models_xcit_large_24_p8_224_float16_training_xpu_accuracy.log 2>&1 popd                              "

set http_proxy=http://proxy.ims.intel.com:911
set https_proxy=http://proxy.ims.intel.com:911
call conda activate "C:/Jenkins/workspace/PTL_Timms_Dynamo_Benchmark\v"
call "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\Build\vcvarsall.bat" x64

pushd C:\Jenkins\workspace\PTL_Timms_Dynamo_Benchmark\pytorch
pip list > "C:\Jenkins\workspace\PTL_Timms_Dynamo_Benchmark\artifacts\timms_pip_list.log" 2>&1
set ZE_PATH=C:\level-zero-win-sdk-1.20.2
python benchmarks\dynamo\timm_models.py --accuracy -d xpu -n10 --backend=inductor --cold-start-latency --training  --float16 --only xcit_large_24_p8_224  --output=C:\Jenkins\workspace\PTL_Timms_Dynamo_Benchmark\artifacts\0\inductor_timm_models_xcit_large_24_p8_224_float16_training_xpu_accuracy.csv >C:\Jenkins\workspace\PTL_Timms_Dynamo_Benchmark\artifacts\0\inductor_timm_models_xcit_large_24_p8_224_float16_training_xpu_accuracy.log 2>&1
popd

`

### Versions

Pytorch 2.11rc
